### PR TITLE
Bump eslint version

### PR DESCRIPTION
**Pull request by KKoukiou**
Autofix warnings: Expected property shorthand

### CODE REVIEW

**Review by martinpitt — DISMISSED** [review]

**Comment by KKoukiou** [comment]
@martinpitt missed three files. Sorry :)

**Review by martinpitt — APPROVED** [review]
No worries. Please land that fast, as it's very prone to bitrotting and conflicting.

File: pkg/lib/cockpit.js [hunk 1]
```diff
@@ -4190,10 +4190,10 @@ function factory() {
         self.get = function get(path, params, headers) {
             return self.request({
                 method: "GET",
-                params: params,
-                path: path,
+                params,
+                path,
```
Comment (cockpituous) [inline pkg/lib/cockpit.js:4194]: These 2 added lines are not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-18417-20230227-173310-602a6f78-fedora-37-devel//Coverage/lcov/github-pr.diff.gcov.html)

File: pkg/lib/cockpit.js [hunk 2]
```diff
@@ -4212,9 +4212,9 @@
 
             return self.request({
                 method: "POST",
-                path: path,
-                body: body,
-                headers: headers
+                path,
+                body,
+                headers
```
Comment (cockpituous) [inline pkg/lib/cockpit.js:4217]: These 3 added lines are not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-18417-20230227-173310-602a6f78-fedora-37-devel//Coverage/lcov/github-pr.diff.gcov.html)

File: pkg/networkmanager/interfaces.js [hunk 3]
```diff
@@ -34,7 +34,7 @@ const _ = cockpit.gettext;
 function show_error_dialog(title, message) {
     const props = {
         id: "error-popup",
-        title: title,
+        title,
```
Comment (cockpituous) [inline pkg/networkmanager/interfaces.js:37]: This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-18417-20230227-173310-602a6f78-fedora-37-devel//Coverage/lcov/github-pr.diff.gcov.html)

File: pkg/networkmanager/interfaces.js [hunk 4]
```diff
@@ -1701,7 +1701,7 @@
                                {
                                    autoconnect: true,
                                    interface_name: iface.Name,
-                                   member_type: member_type,
+                                   member_type,
```
Comment (cockpituous) [inline pkg/networkmanager/interfaces.js:1704]: This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-18417-20230227-173310-602a6f78-fedora-37-devel//Coverage/lcov/github-pr.diff.gcov.html)

File: pkg/storaged/client.js [hunk 5]
```diff
@@ -804,7 +804,7 @@ function legacy_vdo_overlay() {
             }
 
             const v = {
-                name: name,
+                name,
```
Comment (cockpituous) [inline pkg/storaged/client.js:807]: This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-18417-20230227-173310-602a6f78-fedora-37-devel//Coverage/lcov/github-pr.diff.gcov.html)

File: pkg/storaged/dialog.jsx [hunk 6]
```diff
@@ -728,9 +728,9 @@ export const SelectSpaces = (tag, title, options) => {
 
 export const SelectSpace = (tag, title, options) => {
     return {
-        tag: tag,
-        title: title,
-        options: options,
+        tag,
+        title,
+        options,
```
Comment (cockpituous) [inline pkg/storaged/dialog.jsx:733]: These 3 added lines are not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-18417-20230227-173310-602a6f78-fedora-37-devel//Coverage/lcov/github-pr.diff.gcov.html)

File: pkg/storaged/storage-controls.jsx [hunk 7]
```diff
@@ -169,7 +169,7 @@ export class StorageOnOff extends React.Component {
                         .finally(() => { self.setState({ promise: null }) });
             }
 
-            self.setState({ promise: promise, promise_goal_state: val });
+            self.setState({ promise, promise_goal_state: val });
```
Comment (cockpituous) [inline pkg/storaged/storage-controls.jsx:172]: This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-18417-20230227-173310-602a6f78-fedora-37-devel//Coverage/lcov/github-pr.diff.gcov.html)

File: pkg/storaged/utils.js [hunk 8]
```diff
@@ -663,48 +663,48 @@ export function get_active_usage(client, path, top_action, child_action) {
...
+                vgroup,
+                pvol,
                 location: vgroup.Name,
                 actions: get_actions(_("remove from LVM2")),
                 blocking: pvol.FreeSize != pvol.Size
             });
         } else if (vdo) {
             usage.push({
-                level: level,
+                level,
```
Comment (cockpituous) [inline pkg/storaged/utils.js:687]: This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-18417-20230227-173310-602a6f78-fedora-37-devel//Coverage/lcov/github-pr.diff.gcov.html)

File: pkg/storaged/utils.js [hunk 9]
```diff
@@ -663,48 +663,48 @@
...
                 usage: 'vdo-backing',
-                block: block,
-                vdo: vdo,
+                block,
+                vdo,
```
Comment (cockpituous) [inline pkg/storaged/utils.js:690]: These 2 added lines are not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-18417-20230227-173310-602a6f78-fedora-37-devel//Coverage/lcov/github-pr.diff.gcov.html)

File: pkg/storaged/utils.js [hunk 10]
```diff
@@ -663,48 +663,48 @@
...
                 location: vdo.name,
                 blocking: true
             });
         } else if (stratis_pool) {
             usage.push({
-                level: level,
+                level,
```
Comment (cockpituous) [inline pkg/storaged/utils.js:696]: This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-18417-20230227-173310-602a6f78-fedora-37-devel//Coverage/lcov/github-pr.diff.gcov.html)

File: pkg/storaged/utils.js [hunk 11]
```diff
@@ -663,48 +663,48 @@
...
                 usage: 'stratis-pool-member',
-                block: block,
-                stratis_pool: stratis_pool,
+                block,
+                stratis_pool,
```
Comment (cockpituous) [inline pkg/storaged/utils.js:699]: These 2 added lines are not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-18417-20230227-173310-602a6f78-fedora-37-devel//Coverage/lcov/github-pr.diff.gcov.html)

File: pkg/systemd/logDetails.jsx [hunk 12]
```diff
@@ -131,14 +131,14 @@ export class LogEntry extends React.Component {
                         }
                     });
 
-                    this.setState({ entry: entry, loading: false, error: "", problemPath: path, abrtService: service });
+                    this.setState({ entry, loading: false, error: "", problemPath: path, abrtService: service });
                 })
-                .catch(err => this.setState({ entry: entry, loading: false, error: err.toString() }));
+                .catch(err => this.setState({ entry, loading: false, error: err.toString() }));
```
Comment (cockpituous) [inline pkg/systemd/logDetails.jsx:136]: This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-18417-20230227-173310-602a6f78-fedora-37-devel//Coverage/lcov/github-pr.diff.gcov.html)

File: pkg/systemd/logDetails.jsx [hunk 13]
```diff
@@ -148,7 +148,7 @@
                     } else
                         this.setState({ entry: null, loading: false, error: _("Journal entry not found") });
                 })
-                .catch(error => this.setState({ entry: null, loading: false, error: error }));
+                .catch(error => this.setState({ entry: null, loading: false, error }));
```
Comment (cockpituous) [inline pkg/systemd/logDetails.jsx:151]: This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-18417-20230227-173310-602a6f78-fedora-37-devel//Coverage/lcov/github-pr.diff.gcov.html)

File: pkg/systemd/reporting.jsx [hunk 14]
```diff
@@ -330,7 +330,7 @@ class BusWorkflowRow extends React.Component {
                     const message = cockpit.format("reportd task could not be created: $0", (exception.problem || exception.message));
 
                     this.setState({
-                        message: message,
+                        message,
```
Comment (cockpituous) [inline pkg/systemd/reporting.jsx:333]: This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-18417-20230227-173310-602a6f78-fedora-37-devel//Coverage/lcov/github-pr.diff.gcov.html)

File: pkg/users/dialog-utils.js [hunk 15]
```diff
@@ -49,7 +49,7 @@ export function has_errors(errors) {
 function show_error_dialog(title, message) {
     const props = {
         id: "error-popup",
-        title: title,
+        title,
```
Comment (cockpituous) [inline pkg/users/dialog-utils.js:52]: This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-18417-20230227-173310-602a6f78-fedora-37-devel//Coverage/lcov/github-pr.diff.gcov.html)

File: pkg/kdump/config-client-suse.js [hunk 16]
```diff
@@ -182,7 +182,7 @@ export class ConfigFileSUSE extends ConfigFile {
             settings._internal[key].value = value;
         } else {
             if (value)
-                settings._internal[key] = { value: value };
+                settings._internal[key] = { value };
```
Comment (cockpituous) [inline pkg/kdump/config-client-suse.js:185]: This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-18417-20230227-192102-602a6f78-fedora-37-devel//Coverage/lcov/github-pr.diff.gcov.html)

File: pkg/lib/cockpit-components-plot.jsx [hunk 17]
```diff
@@ -195,14 +195,14 @@ function value_ticks(data, config) {
...
-            ticks: ticks,
+            ticks,
             formatter: (val) => config.formatter(val, unit_str, true)[0],
             unit: unit_str,
             max: ticks[ticks.length - 1]
         };
     } else {
         return {
-            ticks: ticks,
+            ticks,
```
Comment (cockpituous) [inline pkg/lib/cockpit-components-plot.jsx:205]: This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-18417-20230227-192102-602a6f78-fedora-37-devel//Coverage/lcov/github-pr.diff.gcov.html)

File: pkg/lib/cockpit.js (hunk 1 above)
Comment (cockpituous) [inline pkg/lib/cockpit.js:4194]: These 2 added lines are not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-18417-20230227-192102-602a6f78-fedora-37-devel//Coverage/lcov/github-pr.diff.gcov.html)

File: pkg/lib/cockpit.js [hunk 18]
```diff
@@ -4190,10 +4190,10 @@
         self.get = function get(path, params, headers) {
             return self.request({
                 method: "GET",
-                params: params,
-                path: path,
+                params,
+                path,
                 body: "",
-                headers: headers
+                headers
```
Comment (cockpituous) [inline pkg/lib/cockpit.js:4196]: This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-18417-20230227-192102-602a6f78-fedora-37-devel//Coverage/lcov/github-pr.diff.gcov.html)

File: pkg/lib/cockpit.js (hunk 2 above)
Comment (cockpituous) [inline pkg/lib/cockpit.js:4217]: These 3 added lines are not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-18417-20230227-192102-602a6f78-fedora-37-devel//Coverage/lcov/github-pr.diff.gcov.html)

File: pkg/networkmanager/interfaces.js (hunk 3 above)
Comment (cockpituous) [inline pkg/networkmanager/interfaces.js:37]: This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-18417-20230227-192102-602a6f78-fedora-37-devel//Coverage/lcov/github-pr.diff.gcov.html)

File: pkg/networkmanager/interfaces.js (hunk 4 above)
Comment (cockpituous) [inline pkg/networkmanager/interfaces.js:1704]: This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-18417-20230227-192102-602a6f78-fedora-37-devel//Coverage/lcov/github-pr.diff.gcov.html)

File: pkg/storaged/client.js (hunk 5 above)
Comment (cockpituous) [inline pkg/storaged/client.js:807]: This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-18417-20230227-192102-602a6f78-fedora-37-devel//Coverage/lcov/github-pr.diff.gcov.html)

File: pkg/storaged/dialog.jsx (hunk 6 above)
Comment (cockpituous) [inline pkg/storaged/dialog.jsx:733]: These 3 added lines are not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-18417-20230227-192102-602a6f78-fedora-37-devel//Coverage/lcov/github-pr.diff.gcov.html)

File: pkg/storaged/storage-controls.jsx (hunk 7 above)
Comment (cockpituous) [inline pkg/storaged/storage-controls.jsx:172]: This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-18417-20230227-192102-602a6f78-fedora-37-devel//Coverage/lcov/github-pr.diff.gcov.html)

File: pkg/storaged/utils.js (hunk 8 above)
Comment (cockpituous) [inline pkg/storaged/utils.js:687]: This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-18417-20230227-192102-602a6f78-fedora-37-devel//Coverage/lcov/github-pr.diff.gcov.html)

File: pkg/storaged/utils.js (hunk 9 above)
Comment (cockpituous) [inline pkg/storaged/utils.js:690]: These 2 added lines are not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-18417-20230227-192102-602a6f78-fedora-37-devel//Coverage/lcov/github-pr.diff.gcov.html)

File: pkg/storaged/utils.js (hunk 10 above)
Comment (cockpituous) [inline pkg/storaged/utils.js:696]: This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-18417-20230227-192102-602a6f78-fedora-37-devel//Coverage/lcov/github-pr.diff.gcov.html)

File: pkg/storaged/utils.js (hunk 11 above)
Comment (cockpituous) [inline pkg/storaged/utils.js:699]: These 2 added lines are not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-18417-20230227-192102-602a6f78-fedora-37-devel//Coverage/lcov/github-pr.diff.gcov.html)

File: pkg/systemd/logDetails.jsx (hunk 12 above)
Comment (cockpituous) [inline pkg/systemd/logDetails.jsx:136]: This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-18417-20230227-192102-602a6f78-fedora-37-devel//Coverage/lcov/github-pr.diff.gcov.html)

File: pkg/systemd/logDetails.jsx (hunk 13 above)
Comment (cockpituous) [inline pkg/systemd/logDetails.jsx:151]: This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-18417-20230227-192102-602a6f78-fedora-37-devel//Coverage/lcov/github-pr.diff.gcov.html)

File: pkg/systemd/reporting.jsx (hunk 14 above)
Comment (cockpituous) [inline pkg/systemd/reporting.jsx:333]: This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-18417-20230227-192102-602a6f78-fedora-37-devel//Coverage/lcov/github-pr.diff.gcov.html)

File: pkg/users/dialog-utils.js (hunk 15 above)
Comment (cockpituous) [inline pkg/users/dialog-utils.js:52]: This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-18417-20230227-192102-602a6f78-fedora-37-devel//Coverage/lcov/github-pr.diff.gcov.html)

File: pkg/kdump/config-client-suse.js (hunk 16 above)
Comment (cockpituous) [inline pkg/kdump/config-client-suse.js:185]: This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-18417-20230227-210817-602a6f78-fedora-37-devel//Coverage/lcov/github-pr.diff.gcov.html)

File: pkg/lib/cockpit-components-plot.jsx (hunk 17 above)
Comment (cockpituous) [inline pkg/lib/cockpit-components-plot.jsx:205]: This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-18417-20230227-210817-602a6f78-fedora-37-devel//Coverage/lcov/github-pr.diff.gcov.html)

File: pkg/lib/cockpit.js (hunk 1 above)
Comment (cockpituous) [inline pkg/lib/cockpit.js:4194]: These 2 added lines are not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-18417-20230227-210817-602a6f78-fedora-37-devel//Coverage/lcov/github-pr.diff.gcov.html)

File: pkg/lib/cockpit.js (hunk 18 above)
Comment (cockpituous) [inline pkg/lib/cockpit.js:4196]: This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-18417-20230227-210817-602a6f78-fedora-37-devel//Coverage/lcov/github-pr.diff.gcov.html)

File: pkg/lib/cockpit.js (hunk 2 above)
Comment (cockpituous) [inline pkg/lib/cockpit.js:4217]: These 3 added lines are not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-18417-20230227-210817-602a6f78-fedora-37-devel//Coverage/lcov/github-pr.diff.gcov.html)

File: pkg/networkmanager/interfaces.js (hunk 3 above)
Comment (cockpituous) [inline pkg/networkmanager/interfaces.js:37]: This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-18417-20230227-210817-602a6f78-fedora-37-devel//Coverage/lcov/github-pr.diff.gcov.html)

File: pkg/networkmanager/interfaces.js (hunk 4 above)
Comment (cockpituous) [inline pkg/networkmanager/interfaces.js:1704]: This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-18417-20230227-210817-602a6f78-fedora-37-devel//Coverage/lcov/github-pr.diff.gcov.html)

File: pkg/storaged/client.js (hunk 5 above)
Comment (cockpituous) [inline pkg/storaged/client.js:807]: This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-18417-20230227-210817-602a6f78-fedora-37-devel//Coverage/lcov/github-pr.diff.gcov.html)

File: pkg/storaged/dialog.jsx (hunk 6 above)
Comment (cockpituous) [inline pkg/storaged/dialog.jsx:733]: These 3 added lines are not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-18417-20230227-210817-602a6f78-fedora-37-devel//Coverage/lcov/github-pr.diff.gcov.html)

File: pkg/storaged/storage-controls.jsx (hunk 7 above)
Comment (cockpituous) [inline pkg/storaged/storage-controls.jsx:172]: This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-18417-20230227-210817-602a6f78-fedora-37-devel//Coverage/lcov/github-pr.diff.gcov.html)

File: pkg/storaged/utils.js (hunk 8 above)
Comment (cockpituous) [inline pkg/storaged/utils.js:687]: This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-18417-20230227-210817-602a6f78-fedora-37-devel//Coverage/lcov/github-pr.diff.gcov.html)

File: pkg/storaged/utils.js (hunk 9 above)
Comment (cockpituous) [inline pkg/storaged/utils.js:690]: These 2 added lines are not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-18417-20230227-210817-602a6f78-fedora-37-devel//Coverage/lcov/github-pr.diff.gcov.html)

File: pkg/storaged/utils.js (hunk 10 above)
Comment (cockpituous) [inline pkg/storaged/utils.js:696]: This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-18417-20230227-210817-602a6f78-fedora-37-devel//Coverage/lcov/github-pr.diff.gcov.html)

File: pkg/storaged/utils.js (hunk 11 above)
Comment (cockpituous) [inline pkg/storaged/utils.js:699]: These 2 added lines are not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-18417-20230227-210817-602a6f78-fedora-37-devel//Coverage/lcov/github-pr.diff.gcov.html)

File: pkg/systemd/logDetails.jsx (hunk 12 above)
Comment (cockpituous) [inline pkg/systemd/logDetails.jsx:136]: This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-18417-20230227-210817-602a6f78-fedora-37-devel//Coverage/lcov/github-pr.diff.gcov.html)

File: pkg/systemd/logDetails.jsx (hunk 13 above)
Comment (cockpituous) [inline pkg/systemd/logDetails.jsx:151]: This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-18417-20230227-210817-602a6f78-fedora-37-devel//Coverage/lcov/github-pr.diff.gcov.html)

File: pkg/systemd/reporting.jsx (hunk 14 above)
Comment (cockpituous) [inline pkg/systemd/reporting.jsx:333]: This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-18417-20230227-210817-602a6f78-fedora-37-devel//Coverage/lcov/github-pr.diff.gcov.html)

File: pkg/users/dialog-utils.js (hunk 15 above)
Comment (cockpituous) [inline pkg/users/dialog-utils.js:52]: This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-18417-20230227-210817-602a6f78-fedora-37-devel//Coverage/lcov/github-pr.diff.gcov.html)

**Pull request merged**